spec/osut_tests_spec.rb

```diff
@@ -2590,8 +2590,12 @@ module M
     expect(core.floorArea).to be_within(TOL).of(149.66)
     core_volume = core.floorArea * 3.05
     expect(core_volume).to be_within(TOL).of(core.volume)
-    expect(attic.volume).to be_within(TOL).of(720.19)
-    expect(attic.floorArea).to be_within(TOL).of(567.98)
+
+    # OpenStudio versions prior to v351 report erroneous volume calculation
+    # results for the attic (798.41 m3).
+    expect(attic.volume).to be_within(TOL).of(720.19) unless v < 350
+    expect(attic.volume).to be_within(TOL).of(798.41)     if v < 350
+    expect(attic.floorArea).to be_within(TOL).of(567.98) # includes overhangs
```

Comment:
The _US DOE SmallOffice Prototype_ attic space has indeed a volume of 720.19 m3. Not sure why SDK versions prior to 3.5.0 are off (a deeper dive into committed fixes seems warranted). These pre-v351 calculations errors trickle down into subsequent tests a bit further in the code.